extensions/amp-brightcove/0.1/test/test-amp-brightcove.js

```diff
@@ -73,8 +75,8 @@ describes.realWin(
       });
     }
 
-    it('renders', () => {
-      return getBrightcove({
+    it('renders', async () => {
+      getBrightcove({
```

Comment:
If you change these test to `async` then you need to use async/await for these methods (`const bc = await getBrightcove();`).

However, I recommend just keeping the tests synchronous and returning the promise.

The reason why the tests were failing with the original setup is because we've added a new promise within `layoutCallback()` that we have to wait for.

Within the `getBrightcove()` helper method in the test, we need to wait for this promise to resolve by using `await macroTask()`:

```
      await element.signals().whenSignal(CommonSignals.LOAD_START);
      // wait for consents 
      await macroTask();

      try {
        fakePostMessage(element, {event: 'ready'});
      } catch (_) {
``` 

With this change, you won't need to change the existing tests.

Reply:
thanks, updated